server/chat-plugins/mafia.ts

```diff
@@ -1596,7 +1596,7 @@ class MafiaTracker extends Rooms.RoomGame {
 
 		if (!player) {
 			if (user.isStaff ||
-				(this.room.auth && user.id in this.room.auth && this.room.auth[user.id] !== '+')
+				(this.room.auth.get(user.id) && this.room.auth.get(user.id) !== '+')
```

Comment:
I would probably have an `auth.above('+')` function, for this which should be a relatively common use-case.